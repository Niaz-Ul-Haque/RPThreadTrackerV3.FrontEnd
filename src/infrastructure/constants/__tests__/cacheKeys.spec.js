import cacheKeys from '../cacheKeys';

describe('object structure', () => {
	it('should have expected values', () => {
		expect(cacheKeys).toMatchSnapshot();
	});
});
