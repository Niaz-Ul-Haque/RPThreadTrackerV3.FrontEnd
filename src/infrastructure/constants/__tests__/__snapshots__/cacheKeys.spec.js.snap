// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`object structure should have expected values 1`] = `
Object {
  "ACCESS_TOKEN": "accessToken",
  "REFRESH_TOKEN": "refreshToken",
  "USE_LIGHT_THEME": "useLightTheme",
}
`;
